cmd/update.go

```diff
@@ -52,6 +49,9 @@ func extractBinary(fileName string) string {
 		panic(err)
 	}
 	tempDir, err := os.MkdirTemp("", "e_grype")
+	if err != nil {
+		panic(err)
```

Comment:
We definitely don't want panics to be in the user's path here consider returning the error up the call stack and modifying the function to `extractBinary(fileName string) (string, error)`